<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollectWindow</class>
 <widget class="QMainWindow" name="CollectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智能温湿度采集控制系统(采集端)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gb_display">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>628</width>
      <height>453</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="gb_data">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>华文琥珀</family>
            <pointsize>32</pointsize>
           </font>
          </property>
          <property name="text">
           <string>智能温湿度采集控制系统</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gb_curTem">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>100</number>
           </property>
           <property name="rightMargin">
            <number>100</number>
           </property>
           <item>
            <widget class="QLabel" name="label_tem">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>当前温度  ：</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="text_tem">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>80</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gb_curHum">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>100</number>
           </property>
           <property name="rightMargin">
            <number>100</number>
           </property>
           <item>
            <widget class="QLabel" name="label_hum">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>当前湿度  ：</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="text_hum">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>80</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
